htcosmos/make_parquet.py

```diff
@@ -65,6 +65,46 @@ def tlog(msg):
     now = datetime.now().strftime('%X.%f')
     print(f'{now} {msg}')
 
+def tlog_flush(msg):
+    now = datetime.now().strftime('%X.%f')
+    print(f'{now} {msg}', flush=True)
+
+# create progress file and tlog the same message, also flush output
+def set_progress(filename, msg):
```

Comment:
docstrings are unnecessary for a lot of these very simple functions, but this one seems worthy of a brief description